experimental/iterators/test/Conversion/IteratorsToStandard/constant.mlir

```diff
@@ -0,0 +1,28 @@
+// RUN: mlir-proto-opt %s -convert-iterators-to-std \
```

Comment:
This test seems superfluous as it could be folded in the print.mlir as a single slightly larger printconstant.mlir ?